apps/dc_tools/odc/apps/dc_tools/s3_to_dc.py

```diff
@@ -110,8 +110,8 @@ def dump_to_odc(
 @request_payer
 @archive_less_mature
 @publish_action
-@click.argument("uri", type=str, nargs=1)
-@click.argument("product", type=str, nargs=1)
+@click.argument("uri", type=str, nargs=-1)
```

Comment:
This `type=str` is ignored, which makes it confusing. I've tested a simple mock script here, and by specifying `nargs=-1`, `uri` will always be a `tuple`.

So, I think remove `type=str`, and rename `uri` to `uris`.